x-pack/plugins/ml/public/alerting/jobs_health_rule/register_jobs_health_alerting_rule.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import { TriggersAndActionsUIPublicPluginSetup } from '../../../../triggers_actions_ui/public';
+import { PluginSetupContract as AlertingSetup } from '../../../../alerting/public';
+import { ML_ALERT_TYPES } from '../../../common/constants/alerts';
+import { MlAnomalyDetectionJobsHealthRuleParams } from '../../../common/types/alerts';
+
+export function registerJobsHealthAlertingRule(
+  triggersActionsUi: TriggersAndActionsUIPublicPluginSetup,
+  alerting?: AlertingSetup
+) {
+  triggersActionsUi.alertTypeRegistry.register({
+    id: ML_ALERT_TYPES.AD_JOBS_HEALTH,
+    description: i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.description', {
+      defaultMessage: 'Alert when anomaly detection jobs experiencing realtime issues.',
+    }),
+    iconClass: 'bell',
+    documentationUrl(docLinks) {
+      return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/machine-learning/${docLinks.DOC_LINK_VERSION}/ml-configuring-alerts.html`;
+    },
+    alertParamsExpression: lazy(() => import('./anomaly_detection_jobs_health_rule_trigger')),
+    validate: (alertParams: MlAnomalyDetectionJobsHealthRuleParams) => {
```

Comment:
Currently, I can select to include all jobs (`*`) and exclude all jobs (`*`). Should there be a guard against that combination?

Reply:
Thanks for the feedback @ymao1! I pushed enhancements for handling excluded jobs in https://github.com/elastic/kibana/pull/106084/commits/8c7b6e669eb2955098fd0d94eb6eac3a6918235d and https://github.com/elastic/kibana/pull/106084/commits/ff5db7e430724f9c22d413cb02efb6522737942c